SQL Assignments/Assign4.sql

```diff
@@ -0,0 +1,359 @@
+-- SOLUTIONS-
+
+-- Write a query to get unique department ID from employee table.
+select distinct DEPARTMENT_ID from employees;
+
+-- Write a query to get all employee details from the employee table order by first name, descending
+select * from employees order by FIRST_NAME;
+
+-- Write a query to get the employee ID, names (first_name, last_name), salary in ascending order of
+-- salary.
+select EMPLOYEE_ID, FIRST_NAME, LAST_NAME, SALARY from employees order by SALARY;
+
+-- Display first name and join date of the employees who is either IT Programmer or Sales Man.
+select FIRST_NAME, HIRE_DATE from employees where JOB_ID = 'SA_MAN' or JOB_ID = 'IT_PROG';
+
+-- Display details of employee with ID 150 or 160.
+select * from employees where EMPLOYEE_ID='150' or EMPLOYEE_ID='160';
+
+-- Display first name, salary, commission pct, and hire date for employees with salary less than 10000.
+select FIRST_NAME, SALARY, COMMISSION_PCT from employees where SALARY < 10000;
+
+-- Display employees where the first name or last name starts with S.
+select FIRST_NAME, LAST_NAME from employees where FIRST_NAME like 'S%' or FIRST_NAME like '_S';
+
+-- Display details of jobs in the descending order of the title.
+select * from jobs order by JOB_TITLE DESC;
+
+-- Display details of the employees where commission percentage is null and salary in the range 5000
+-- to 10000 and department is 30.
+select * from employees where COMMIsSION_PCT is null and SALARY>5000 and SALARY>10000 and DEPARTMENT_ID='30';
+
```

Comment:
Please check the question and update the query

Reply:
Done